src/test/resources/features/Demo.feature

```diff
@@ -2,10 +2,10 @@ Feature: Create a Dashboard in Trello
 
```

Comment:
We should be to create specific feature file for each test, Demo.feature is not good nor descriptive name